pynamodb/constants.py

```diff
@@ -245,6 +252,53 @@
     BETWEEN: 'between'
 }
 
+# each transaction can only support 10 actions at a time
+# https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/Limits.html#limits-dynamodb-transactions
+TRANSACT_ITEMS_LIMIT = 10
```

Comment:
Sure :0 Should I also remove the client request token validation (that its a string/less than 37 characters)? I originally added that because my team got pretty confused about the source of a write failure that ultimately ended up being that the request token they were using was too long

Reply:
If the message that the API gives is unclear then happy to have validation here. This is just something that could easily change in a future DynamoDB release, so better if we don't hardcode validation logic.

Reply:
That's a fair point :0 I don't think its confusing enough to make it a blocker if dynamo changes in the future